src/test/regress/expected/gporca.out

```diff
@@ -13458,15 +13458,15 @@ and first_id in (select first_id from mat_w);
          SubPlan 1  (slice3; segments: 3)
            ->  Materialize  (cost=0.00..4.69 rows=3 width=4)
                  Output: material_test.first_id
-                 ->  Broadcast Motion 3:3  (slice1; segments: 3)  (cost=0.00..4.65 rows=3 width=4)
+                 ->  Broadcast Motion 3:3  (slice1; segments: 3)  (cost=0.00..4.65 rows=8 width=4)
```

Comment:
@dnskvlnk, take a look two lines above. The Broadcast emits ~8 tuples after your changes, but the Materialize still emits ~3 tuples as far as estimation was copied directly from Seq Scan for this node. It's not enough to multiply rows out for Broadcast only.

Reply:
Fixed with `broadcast_motion_mutator`